js/jsonld.js

```diff
@@ -1646,6 +1646,30 @@ jsonld.cache = {
 };
 
 /**
+ * Accept header.
+ */
+jsonld.acceptHeader = 'application/ld+json, application/json';
```

Comment:
I recommend we just take `acceptHeader` off of the public API for now (off of  `jsonld.`) and make a private module defaults var:

```js
var _defaults = {
  headers: {
    accept: 'application/ld+json, application/json'
  }
};
```

That way we move towards something we can expose in the future but we don't have to decide on exactly what it's shape is since that's not the point of this PR right now.